scalafmt-tests/src/test/scala/org/scalafmt/util/GitOpsTest.scala

```diff
@@ -37,7 +37,7 @@ class GitOpsTest extends fixture.FunSuite {
 
   def touch(name: String = Random.alphanumeric.take(10).mkString)(
       implicit ops: GitOpsImpl): AbsoluteFile = {
-    val f = File.createTempFile(name, ".ext", ops.workingDirectory.jfile)
+    val f = File.createTempFile(name, ".ext", ops.rootDir.get.jfile)
```

Comment:
Yeah this was my original plan, but I wanted to avoid dirtying the dev's working dir if anything went wrong. 

But if something goes wrong it isn't the end of the world.